scripts/nn/util.dml

```diff
@@ -200,3 +200,19 @@ unpad_image = function(matrix[double] img_padded, int Hin, int Win, int padh, in
   }
 }
 
+threshold = function(matrix[double] X, double thresh)
+    return (matrix[double] out) {
+  /*
+   * Computes an indicator matrix with values in (0, 1) depending on whether or not the values in X
```

Comment:
Yeah, that would be awesome.  I have a bunch of style guidelines that I've been implicitly following, but I haven't written them up.  2-space indentation, 4-space continuation lines, 72-character docstrings, the formatting of inputs & outputs in the docstring, 100-character code line-length, the function & return declaration formatting....